src/main/kotlin/no/nav/syfo/dialogmote/tilgang/DialogmoteTilgangService.kt

```diff
@@ -89,14 +89,16 @@ class DialogmoteTilgangService(
             if (krrEnabled) {
                 true
             } else {
-                val kontaktinfo = kontaktinformasjonClient.kontaktinformasjonWithOBO(personIdentNumber, token, callId)
-                val isDigitalVarselEnabled = kontaktinfo?.kontaktinfo?.isDigitalVarselEnabled(personIdentNumber)
-                if (isDigitalVarselEnabled == false) {
+                val isDigitalVarselEnabled = kontaktinformasjonClient.isDigitalVarselEnabled(
```

Comment:
Kan evt refaktorere det inni else her til en metode ala `checkDigitalVarselEnabled(): boolean` så man kan forenkle til` return krrEnabled || checkDigitalVarselEnabled()`. Ikke noe must, men bare for å kvitte seg med litt mer duplisering :)

Reply:
Bedre nå?